Note added for https://github.com/godot-rust/gdext/pull/365 (#109)
godot-rust/book: src/gdext/index.md

3. **Always-on**  
   

>    There is no differentiation between "tool" and "normal" scripts anymore; Rust logic runs as soon as the Godot editor launches, but gdext explicitly changes this behaviour, by default all virtual callbacks (ready, process etc.) are not invoked **in editor mode**, this behaviour can be reverted with a configuration, while implementing the [ExtensionLibrary][extension_library_doc] trait.

Reviewer: Maybe use some full stops, otherwise this sentence is very long. You can then split some lines, too.
Also, we use American english ("behavior" instead of "behaviour"). Any code symbols I would put inside backticks. 🙂 

```suggestion
   There is no differentiation between "tool" and "normal" scripts anymore, as it was the case in GDNative. Rust logic runs as soon as 
   the Godot editor launches, but gdext explicitly changes this behavior. By default, all virtual callbacks (`ready`, `process` etc.)
   are not invoked **in editor mode**. This behavior can be configured with the [`ExtensionLibrary`][extension-library-doc] trait.
```

[features]: https://github.com/godot-rust/gdextension/issues/24
[issue #66231]: https://github.com/godotengine/godot/issues/66231

> [extension_library_doc]: https://godot-rust.github.io/docs/gdext/master/godot/init/trait.ExtensionLibrary.html#method.editor_run_behavior

Reviewer: Minor nitpick for consistency:
```suggestion
[extension-library-doc]: https://godot-rust.github.io/docs/gdext/master/godot/init/trait.ExtensionLibrary.html#method.editor_run_behavior
```
